src/libYARP_dev/src/IVisualParamsImpl.cpp

```diff
@@ -202,12 +208,11 @@ bool Implement_RgbVisualParams_Parser::respond(const yarp::os::Bottle& cmd, yarp
                     ret = iRgbVisual->getRgbIntrinsicParam(params);
                     if(ret)
                     {
-                        yarp::os::Bottle params_b;
                         response.addVocab(VOCAB_RGB_VISUAL_PARAMS);
                         response.addVocab(VOCAB_INTRINSIC_PARAM);
                         response.addVocab(VOCAB_IS);
-                        ret &= Property::copyPortable(params, params_b);  // will it really work??
-                        response.append(params_b);
+                        yarp::os::Bottle& tmp=response.addList();
+                        ret &= Property::copyPortable(params, tmp);
```

Comment:
The `&=`  form is pretty used as a shortcut for `a = a && b`, as nearly all functions in the controlBoardWrapper use it (and in my opinion `&=`  is more readable then the extended version).
I'd not enforce a standard for this and let the writer choose which version he/she prefer.